### ISSUE TITLE
As a user I want to quickly assign myself to tasks from the Kanban view

### ISSUE THREAD

**Issue by zanona** [ZenHubIO/support#1291]
Hey guys, I often need to assign/de-assign myself from tasks from the Kanban view which I leave open constantly on a second display to have a better notion of what's next. 

However, the __current process__ is quite burdensome with the following:

1. Click on the task/story
2. Wait for it to load (sometimes it can take a while)
3. Click the assignee's cogwheel (in case there are others assigned)
4. Wait for it to load members/contributors (sometimes it also take a while)
5. Select myself
6. Close the story modal

__What I would like to have is:__

1. click the profile picture under the story I want to work on
2. it will just assign/de-assign myself as I click

![](https://vignette.wikia.nocookie.net/glee/images/c/ca/Simples.gif/revision/latest?cb=20150220233403)

**Comment by kendramoroz** [comment]
@zanona agreed! We're actually looking into a quick action menu option for the Board to make this workflow easier. Looping in @brianleung11-3 the designer on the team working on this and bumping over to discussion while we dig into the details 👍
